core/kernel/Makefile

```diff
@@ -0,0 +1,11 @@
+FLATBUFFERS := $(shell find -P flatbuffers -name "*.fbs" )
+
+generate_flatbuffers:
```

Comment:
Is there a way to turn this into a rule? So that make figures out when to generate a flatbuffer from a .fbs file?

Reply:
Files will be generated as a pre-build step. Locally this means that the developer will have to run this target in order to have to local development setup.

Then after making changes to the FlatBuffers schema, the target will have to be generated again. Possibly, to improve the development experience, it will be better to have a rule to automatically generate the files when schema changes, I'll think about it.

On CI, this means that before running test/checks/packaging, it will be required to generate FlatBuffers.

Reply:
I've discovered that Cargo allows having pre-build code executed. I have added [build.rs](https://github.com/polypoly-eu/polyPod/blob/c295ca6a7cae3ca0e1afe92e09255ee565e4714a/core/kernel/build.rs), which will contain any code that needs to be executed before building Kernel.

For flatbuffers, `build.rs` will re-generate the FlatBuffers when a change has been made.